tests/test_api_jws.py

```diff
@@ -303,6 +303,12 @@ def test_invalid_crypto_alg(self, jws, payload):
         with pytest.raises(NotImplementedError):
             jws.encode(payload, 'secret', algorithm='HS1024')
 
+    @pytest.mark.skipif(has_crypto, reason='Tests better errors if crypography not installed')
```

Comment:
I think this would be better if it just said "Scenario requires that cryptography not be installed"